posts/2022-12-14-android-ndk-update.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Updating the Android NDK in Rust <version>"
+author: Android Platform Team
+description: "Modernizing Android support in Rust"
+---
+
+We are pleased to announce that Android platform support in Rust will be
+modernized in Rust <version> as we update the target NDK from r17 to r25.  As a
+consequence the minimum supported API level will increase from 15 (Jelly Bean)
+to 19 (KitKat).
+
+In NDK r23 Android switched to using LLVMs `libunwind` for all architectures.
+This meant that
+1. If a project were to target NDK r23 or newer with previous versions of Rust
+   [a workaround](https://github.com/rust-lang/rust/pull/85806#issuecomment-1096266946)
+   would be required to redirect attempts to link against `libgcc` to instead
+   link against `libunwind`.  Following this update this workaround will no
+   longer be necessary.
+2. If a project targets NDK r22 or older it will need to be updated to target
```

Comment:
Minor nitpick when I read this all the way through: Would "uses" and "use" be better than "targets"/"target" here? I generally feel like target is used more for API level (like the 19/KitKat mentioned above) or architecture (like aarch64).

Reply:
Done